From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-kmemleak-split-__create_object-into-two-functions-checkpatch-fixes
Date: Wed Oct 18 08:40:15 AM PDT 2023


ERROR: "foo * bar" should be "foo *bar"
#39: FILE: mm/kmemleak.c:626:
+static struct kmemleak_object * __alloc_object(gfp_t gfp)

total: 1 errors, 0 warnings, 99 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/mm-kmemleak-split-__create_object-into-two-functions.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Catalin Marinas <catalin.marinas@arm.com>
Cc: Kefeng Wang <wangkefeng.wang@huawei.com>
Cc: Liu Shixin <liushixin2@huawei.com>
Cc: Patrick Wang <patrick.wang.shcn@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/kmemleak.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/kmemleak.c~mm-kmemleak-split-__create_object-into-two-functions-checkpatch-fixes
+++ a/mm/kmemleak.c
@@ -623,7 +623,7 @@ static noinline depot_stack_handle_t set
 	return trace_handle;
 }
 
-static struct kmemleak_object * __alloc_object(gfp_t gfp)
+static struct kmemleak_object *__alloc_object(gfp_t gfp)
 {
 	struct kmemleak_object *object;
 
_
